lib/cc/services/github_pull_requests.rb

```diff
@@ -87,8 +86,7 @@ def presenter
   def update_status_error
     update_status(
       "error",
-      "Code Climate encountered an error while attempting to analyze this " +
-        "pull request."
+      message
```

Comment:
@pbrisbin okay that's fine by me. I see some reasons supporting both places as the site of setting a `message_default` - maybe it makes sense to have messages get set in one place, so `finalizer` should worry about the messages and `services` can just eat them. But on the other hand, I think having a default message on `services` side is fine. 

Generally, agree that it seems a little confusing that the presenter is used for some messages and not others (and the method is named `success_message` even when the status is failure :p ). But punting sounds ideal to me. :) Keep changes separate.